/cgroup/cpu.stat doesn't exist any more.  Change over to cgroup.procs.

Index: lxc-0.7.5/src/lxc/cgroup.c
===================================================================
--- lxc-0.7.5.orig/src/lxc/cgroup.c
+++ lxc-0.7.5/src/lxc/cgroup.c
@@ -248,7 +248,7 @@ lxc_mount_cgroup(void)
 			goto out;
 		}
 	}
-	if (access("/cgroup/cpu.stat", F_OK)) {
+	if (access("/cgroup/cgroup.procs", F_OK)) {
 		if (mount("cgroup", "/cgroup", "cgroup", 0, NULL)) {
 			err = -1;
 			SYSERROR("Unable to mount /cgroup");
